docs/apache-airflow/concepts/deferring.rst

```diff
@@ -55,7 +55,7 @@ Writing Deferrable Operators
 
 Writing a deferrable operator takes a bit more work. There are some main points to consider:
 
-* Your Operator must defer itself based on a Trigger. If there is a Trigger in core Airflow you can use, great; otherwise, you will have to write one.
```

Comment:
I thought "based on" was a little less clear... based on in what sense?   Based on could imply that the deferral is in _response_ to a trigger, that some how the trigger comes before the deferral.  By using "with" instead, I think it's slightly clearer that that you must include a trigger instance in your deferral, i.e. as part of the deferral, i.e. within the deferral exception.